### PR TITLE
chore(model-api-gen): add comment about "updateDependencies" task from NPM packages

### PR DESCRIPTION
**Pull request by odzhychko**
Documenation of build files for future developers and myself 😆 

## To be verified by reviewers

* [ ] Relevant public API members have been documented
* [ ] Documentation related to this PR is complete
  * [ ] Boundary conditions are documented
  * [ ] Exceptions are documented
  * [ ] Nullability is documented if used
* [ ] Touched existing code has been extended with documentation if missing
* [ ] Code is readable
* [ ] New features and fixed bugs are covered by tests

### CODE REVIEW

File: model-api-gen-gradle-test/vue-integration/build.gradle.kts [hunk 1]
```diff
@@ -5,6 +5,25 @@ plugins {
     alias(libs.plugins.node)
 }
 
+/**
+ * This task is needed because the listed packages have been rebuilt (locally or CI).
+ * As a result, the verification hashes in package-lock.json are out of date.
+ * Reinstalling them will update their hashes.
+ * This issue only arises because we are using the packaged libraries.
+ *
+ * We use packaged libraries for the following reason:
```
Comment (mhuster23) [inline model-api-gen-gradle-test/vue-integration/build.gradle.kts:14]: I suggest moving this below the paragraph starting at L16 and mark it with (b) so it's clearer, that we use b out of those two options.

Comment (odzhychko) [reply model-api-gen-gradle-test/vue-integration/build.gradle.kts]: Done

File: model-api-gen-gradle-test/vue-integration/build.gradle.kts [hunk 2]
```diff
@@ -5,6 +5,25 @@ plugins {
...
+ *
+ * If you want to install a local package, it makes a difference whether you
+ * (a) link the folder with the package.json and the built files,
+ * (b) or pack it into a `tgz' using `npm pack'.
+ *
+ * Packaged local packages (b) behave like packages pulled from NPM.
+ * In case (a), node doesn't behave like packages pulled by NPM.
+ * Such packages will be loaded more than once if they are imported from different locations.
+ * This is relevant when working with global stuff.
```
[the remark this answers is not in the document]
Comment (odzhychko) [reply model-api-gen-gradle-test/vue-integration/build.gradle.kts]: Done
Also added even more explenation from my side.

**Pull request merged**